tony-examples/tony-in-gcp/README.md

```diff
@@ -0,0 +1,524 @@
+![img](https://i.imgur.com/7vEjSTC.png=100x100)
+
+## Introduction
+
+Cloud [Dataproc](https://cloud.google.com/dataproc/) is a managed Spark and Hadoop service that lets you take advantage of open source data tools for batch
+processing, querying, streaming, and machine learning. Cloud Dataproc automation helps you create clusters quickly, 
+manage them easily, and save money by turning clusters off when you don't need them. With less time and money spent on 
+administration, you can focus on your jobs and your data.
+
+The goal of this document is to provide the instructions required to install a Hadoop cluster for LinkedIn Open source 

```

Comment:
The goal should be launching a distributed tensorflow job? The documentation will provide the instructions XXX